liota/lib/transports/web_socket.py

```diff
@@ -56,6 +56,7 @@ def connect_soc(self):
             self.WebSocketConnection(self.url, False)
         except Exception:
             log.exception("WebSocket exception, please check the WebSocket address and try again.")
+            raise Exception("WebSocket exception, please check the WebSocket address and try again.")
```

Comment:
Liotad will never stop, the package won`t be loaded it is the general case and is tested in this scenario. And it's better to catch the exception and print the message around checking the Websocket address rather than just throwing a general exception.